vars/main.yml

```diff
@@ -23,6 +23,14 @@ core:
   upf:
     ip_prefix: "192.168.252.0/24"
     iface: "access"
+    # af_packet or dpdk
+    # when using 'dpdk' mode please make sure atleast two VF devices are
+    # created out of 'data_iface' and both these devices attached to the
+    # vfio-pci driver
```

Comment:
Is not this "transparent" to the user? That is, this is automatically taken care of by Ansible, no? If so, why do we need this text here?

Reply:
yes, it is taken care automatically. I have updated it now.